doc/metric-yaml.rst

```diff
@@ -0,0 +1,199 @@
+.. currentmodule:: lsst.validate.base
+
+.. _validate-base-metric-yaml:
+
+###########################################
+Defining Metrics and Specifications in YAML
+###########################################
+
+*Metrics* are scalar quantities that can be measured and monitored.
+For example, ``validate_drp`` is designed to measure metrics from :lpm:`17`, the LSST Science Requirements document.
+:lpm:`17`, defines metrics that quantify photometric and astrometric measurement accuracy in the LSST Science Pipelines.
+
+Each metric can be accompanied by several *specification* levels.
+Specifications are thresholds of a metric that define success or give some indication of algorithm development progress.
+:lpm:`17`, for example, defines 'minimum,' 'design,' and 'stretch' specifications for each metric.
+
+Defining metrics and specifications in a YAML_ file allows you to separate science configuration from code, while still keeping metrics accessible from the code and versioned in Git.
+
+This page describes the schema describing for metrics and specifications in YAML_.
+
+.. seealso::
+
+   :ref:`validate-base-using-metrics`.
+
+.. _validate-base-metric-objects-yaml:
+
+Metric Objects in YAML
+======================
+
+Each metric is a separate key-value object in a YAML file.
+For example, here's the AM1 metric's definition:
+
+.. code-block:: yaml
+
+   AM1:
+     reference:
+       doc: LPM-17
+       url: http://ls.st/lpm-17
+       page: 23
+     description: >
```

Comment:
Good point. It's a YAML way of saying that the next line is a multi-line paragraph that should be parsed into a single line. I'm adding a note in the docs about this because it's a non-obvious YAML feature.